test/TransactionHandler/handle.test.ts

```diff
@@ -0,0 +1,65 @@
+import TransactionHandler from '../../src/TransactionHandler';
+import StakeRequestedHandler from '../../src/handlers/StakeRequestedHandler';
+import SpyAssert from '../utils/SpyAssert';
+import StubData from '../utils/StubData';
+
+import chai = require('chai');
+import chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const { assert } = chai;
+
+import sinon = require('sinon');
+
+describe('TransactionHandler.handle()', () => {
+  const bulkTransactions = {
+    stakeRequesteds: [
+      {
+        __typename: 'StakeRequested',
+        amount: '0',
+        beneficiary: '0x79376dc1925ba1e0276473244802287394216a39',
+        gasLimit: '2',
+        gasPrice: '1',
+        gateway: '0x4e4ea3140f3d4a07e2f054cbabfd1f8038b3b4b0',
+        id: '0xa80c3db5089412e553b3b4defc3b3759f56b3a77257be6940251a7a05b5c4fec-0',
+        nonce: '1',
+        stakeRequestHash: '0xdc67e167a7dd111e4f2c27796ceb89955bb68b995eef3a84aa86b38a5f7cd22c',
+        staker: '0x79376dc1925ba1e0276473244802287394216a39',
```

Comment:
Addresses and hashes are actual data. Let us use data that can be identified as the dummy
```
{
        __typename: 'StakeRequested',
        amount: '0',
        beneficiary: '0x0000000000000000000000000000000000000001',
        gasLimit: '2',
        gasPrice: '1',
        gateway: '0x0000000000000000000000000000000000000002',
        id: '0x0000000000000000000000000000000000000000000000000000000000000002-0',
        nonce: '1',
        stakeRequestHash:'0x0000000000000000000000000000000000000000000000000000000000000001',
        staker: '0x0000000000000000000000000000000000000003',
}
```

Reply:
why?

Reply:
This is dummy data and not actual data.

Reply:
This is also dummy, why do you think it's actual.

Reply:
By seeing I can identify that its a dummy data. 
Otherwise, I will expect that there is some significance of particular value used and why that value was selected. We had this discussion in the past. 

In this example, if we are going to validate the stake request hash then it will not be a dummy value.
Anyways it's not that critical. Will add it to developer guidelines.